packages/app/src/connecting/components/connecting.test.tsx

```diff
@@ -103,18 +112,86 @@ describe("`BackupDeviceFlow` component", () => {
     })
   })
 
-  describe("when `noModalsVisible` is set to `false` and unlocked to `false`", () => {
-    const extraProps: Partial<Props> = {
-      noModalsVisible: false,
-      unlocked: false,
-    }
+  describe("`CriticalBatteryLevelModal` and `PasscodeModal` components", () => {
+    test("when unlocked, noModalsVisible and criticalBatteryLevel are false, then both modals are hidden", () => {
```

Comment:
🟢 for tests that differ only by parameters, you can use test.each  
https://jestjs.io/blog/2018/05/29/jest-23-blazing-fast-delightful-testing#jest-each

Reply:
Thanks, didn't know :)